src/Html.php

```diff
@@ -3383,7 +3383,7 @@ class='dropdown_tooltip {$param['link_class']}'";
         if (!is_null($param['url'])) {
             $js .= "
                 ajax: {
-                    url: '" . htmlescape($CFG_GLPI['root_doc'] . $param['url']) . "',
+                    url: '" . URL::sanitizeURL($CFG_GLPI['root_doc'] . $param['url'])  . "',
```

Comment:
`URL::sanitizeURL()` is made to prevent malicious links based on user input, but is not handling the "escaping" required here. The correct "escaping" function for a JS variable is `json_encode` (or `|e('js')` in Twig context).

```suggestion
                    url: " . json_encode($CFG_GLPI['root_doc'] . $param['url'])  . ",
```